src/client/componentsMealsharing/ReviewSpecific.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import "./Meal.css";
+
+export const ReviewSpecific = ({ match }) => {
+  const id = match.params.id;
+  const [fetchItem, setFetchItem] = useState([]);
```

Comment:
again should have a more descriptive name.

Reply:
also why is it an array when there is only one item fetched?

Reply:
yes,i used to use a map, but here we do not need that array.